fab/inventory/l10k

```diff
@@ -0,0 +1,61 @@
+[l10kserver1]
+10.162.36.254
+
+#[l10kbackup1]
+#ip_backup_server
```

Comment:
true. but also, even if there no VPN, if both servers are plugged into the same switch, and the both have two NICs (as servers usually do) we can use one of the NICs for 10.0.0.x and use it as a LAN, also if in the same switch, the switch should only transfer from one machine to the other, so same security as VPN.  Still, I'd prefer to have the both in a LAN and have different firewall rules for LAN and WAN.